bodhi/server/models.py

```diff
@@ -1483,6 +1487,20 @@ def unpush(self, koji):
                         self.nvr, tag, release.candidate_tag))
                 koji.moveBuild(tag, release.candidate_tag, self.nvr)
 
+    def is_latest(self) -> bool:
+        """Check if this is the latest build available in the stable tag."""
+        koji_session = buildsys.get_session()
+        builds = koji_session.getLatestBuilds(
```

Comment:
Will this return every build that is not in stable yet?

Reply:
No that will return the latest builds in the stable tag (ie f31) for the specified package name. It is a more efficient way to know if there is a build in the stable tag, rather than getting all the builds tagged with f31.

Reply:
Nice to know